src/test/java/edu/unc/lib/boxc/migration/cdm/CdmExportCommandIT.java

```diff
@@ -121,6 +124,40 @@ public void noPasswordTest() throws Exception {
         assertOutputContains("Must provided a CDM password");
     }
 
+    @Test
+    public void lessThanMinPageSizeTest() throws Exception {
+        Path projPath = createProject();
+
+        String[] args = new String[] {
+                "-w", projPath.toString(),
+                "export",
+                "--cdm-url", cdmBaseUrl,
+                "-p", PASSWORD,
+                "-n", "0"};
+        executeExpectFailure(args);
+
+        MigrationProject project = MigrationProjectFactory.loadMigrationProject(projPath);
+        assertFalse("Description folder should not be created", Files.exists(project.getExportPath()));
```

Comment:
I think the message on this assertion is off, should probably be "Export folder must not exist". Same with the assertion in the other new tests